### ISSUE TITLE
[Bug]: Responsive Webdesign

### ISSUE THREAD

**Issue by lvlcn-t** [2223-Project-1/OmegaPass#67]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

See at Anything else.

### Expected Behavior

Everything should look just as on desktop only compressed.

### Steps To Reproduce

1. Go on https://omegapass.de
2. Open DevTools
3. Click on the "Toggle Device Toolbar" Button
4. Select any smartphone (preferably with a small screen)

### Environment Info

```markdown
- Device: iPhone SE
- OS: DevTools
- Browser: Brave
- Version: 1.51.114
```

## Device: iPhone SE
![image](https://github.com/2223-Project-1/OmegaPass/assets/75443136/3aff8bac-a8b2-4e59-90f9-23b11ed1d29d)

**Comment by techmaved** [comment]
Responsive Design wasn't our priority, but I do what I can do till the #34

**Comment by lvlcn-t** [comment]
> Responsive Design wasn't our priority, but I do what I can do till the #34

Yeah it's a minor thing, but it'd be nice if we could say in our presentation that everyone can use it on any device 🙂

**Comment by techmaved** [comment]
Made a quick [implementation](https://github.com/2223-Project-1/OmegaPass/tree/feat/%2367). When you have lots of entries you need to scroll. We could devide them into pages.

**Comment by techmaved** [comment]
> > Made a quick [implementation](https://github.com/2223-Project-1/OmegaPass/tree/feat/%2367). When you have lots of entries you need to scroll. We could devide them into pages.
> 
> Good first implementation but I thought more of the concept of #29 (bottom of the picture)

Oh yeah I forgot
